wca-guidelines.md

```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler must verify there are no duplicate scrambles.
```

Comment:
it seems a bit odd to list a "must regulation" as a "recommendation". Perhaps "clarification" would be better suited?

Reply:
Indeed, this should not be a recommendation. It will be fixed :)